plugins/module_utils/dnac.py

```diff
@@ -68,8 +71,10 @@ def __init__(self, module):
         # Check if 'dnac_log_level' in the playbook params. If available,
         # convert it to uppercase; otherwise, set it to 'INFO'
         self.dnac_log_level = dnac_params.get("dnac_log_level", "INFO").upper()
+        self.dnac_log_file_path = dnac_params.get("dnac_log_file_path", "dnac.log")
+        self.dnac_logs_append = dnac_params.get("dnac_logs_append", "True")
 
-        log(str(dnac_params))
+        self.log(str(dnac_params))
```

Comment:
Can we change this log message as per our guidelines?

For example
self.log("Dnac params: {0}".format(....), "DEBUG")